jenkins-slaves/jenkins-slave-ansible/Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM openshift3/jenkins-slave-base-rhel7:latest
+FROM openshift/jenkins-slave-base-centos7
 
-USER root
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
 
-RUN yum install -y --setopt=tsflags=nodocs \
-      --disablerepo=* --enablerepo=rhel-7-server-rpms --enablerepo=rhel-7-server-extras-rpms \
-      https://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/ansible-2.5.8-1.el7.ans.noarch.rpm && \
-    rpm -V ansible && \
-    yum clean all -y && \
-    rm -rf /var/cache/yum
+RUN INSTALL_PKGS="java-1.8.0-openjdk-devel.x86_64 git tree vim unzip curl" && \
+    x86_EXTRA_RPMS=$(if [ "$(uname -m)" == "x86_64" ]; then echo -n java-1.8.0-openjdk-devel.i686 ; fi) && \
+    yum install -y centos-release-scl-rh && \
+    yum install -y --enablerepo=centosplus $INSTALL_PKGS $x86_EXTRA_RPMS && \
+    yum clean all -y
+
+RUN yum install -y epel-release && \
```

Comment:
Please merge this `RUN` statement with the above one to reduce the number of layers in the image.